README.md

```diff
@@ -170,6 +170,16 @@ If you wish to make larger changes we generally recommend [forking](https://help
 
 ### Build Hets using Stack
 * [Install Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade) (use the generic Linux option if you are on Ubuntu).
+* Install build- and GUI-dependencies
+  * Ubuntu:
+    ```
+    sudo apt-get install libglib2.0-dev libcairo2-dev libpango1.0-dev libgtk2.0-dev libglade2-dev libncurses-dev
+    ```
```

Comment:
Ehmm, you are mixing up two different things, i.e. the "new build" which bypasses the system, and the proper build which relies on the system. It is not a question, whether to drop haskeline (terminfo), but how they are build and used by ghc when it finally links the binary. If you bypass the system, there is no haskeline and it probably gets build with given defaults, which might be with libncurses directly or indirectly via terminfo, which than requires the additional dev packages. If one does not bypass the system, the haskeline comming with the ghc package gets used: this is not linked or required to link against libncurses or uses a terminfo, which itself does not require libncurses (or may require it, but than this is solved wrt. the terminfo module and its given package dependencies). So it shouldn't surprise one, that for different ways of building stuff there are different dependencies .... Wrt. ubuntu packages, those have to go into debian/control, with the hybrid/system bypassing build it probably needs to be persisted  somewhere else ...